### ISSUE TITLE
preceding-sexp is deprecated in emacs 25

### ISSUE THREAD

**Issue by shlomiv** [hchbaw/eval-sexp-fu.el#4]
Hey, its not yet a bug, but you should probably [move away from preceding-sexp](https://github.com/hchbaw/eval-sexp-fu.el/blob/master/eval-sexp-fu.el#L399) for emacs 25.

```
Warning (bytecomp): ‘preceding-sexp’ is an obsolete function (as of 25.1); use ‘elisp--preceding-sexp’ instead.
```

**Comment by stardiviner** [comment]
Add an if condition to use different version function?

**Comment by stardiviner** [comment]
Now Emacs 25.1 is released. I think really should update now.

**Comment by hchbaw** [comment]
It could be resolved by #3 and #5, so please reopen this issue if it does not work.

Anyway thank you very much for your reports.
